### PR TITLE
cap3

### PR DESCRIPTION
**Pull request by Mataivic**
* [x] I have read the guidelines above.
* [x] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).
@bioconda/core

### CODE REVIEW

File: recipes/cap3/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+package:
+  name: cap3
+  version: '10.2011'
+
+source:
+  fn: cap3.linux.x86_64.tar # [linux]
+  url: http://seq.cs.iastate.edu/CAP3/cap3.linux.x86_64.tar # [linux]
```
Comment (bgruening) [inline recipes/cap3/meta.yaml]: Can you try to compile it from source? Precompiled binaries are not guaranteed to run on older machines.

Comment (Mataivic) [inline recipes/cap3/meta.yaml]: Hello, I tried, but only the executable files are available.

(n.b : I'm a beginner bioinformatician here, sorry if I'm not answering correctly)

Comment (Mataivic) [inline recipes/cap3/meta.yaml]: Ok, I will try to get in touch in touch with them as soon as I'm back to work tomorrow. But I already tried to contact one of them some days ago and I did not have any answer.

Comment (Mataivic) [inline recipes/cap3/meta.yaml]: Hello,

Still no answer from the authors ... Is it still okay ?

Comment (bgruening) [inline recipes/cap3/meta.yaml]: Thanks for trying!

**Pull request merged**